cmd/livepeer_cli/wizard_transcoder.go

```diff
@@ -66,13 +67,16 @@ func (w *wizard) promptOrchestratorConfig() (float64, float64, int, int, string)
 	fmt.Printf("Enter the price for %d pixels in Wei (required) ", pixelsPerUnit)
 	pricePerUnit := w.readDefaultInt(0)
 
-	addr := myHostPort()
+	if w.offchain {
```

Comment:
I think that this `if` does not make sense, because you can never run this function in the offchain mode. Orchestrator activation and setting orchestrator config is only for on-chain.

However, I think you need another condition here.
1. If `orch.ServiceURI` is already set (orchestrator is already registered on-chain), then use `orch.ServiceURI`. 
2. Otherwise, use `myHostPort()`

Reply:
Got it, that makes more sense. I was thinking there might be some use cases for off-chain where `myHostPort` needs  is used as-is. Will update PR.

Reply:
Addressed all comments. Also added a condition to not allow orchestrator config settings to be changed if off-chain is detected based on your comments. 

Only outstanding issue was that test.sh fails locally for unrelated reasons - likely related to clang compiler that gets used by default on my Mac M1. The tests executed in this PR all pass so this should be OK to merge. If it's alright, I can follow up with a fix for test.sh failures on Macs as part of a separate ticket. 
```
/opt/homebrew/Cellar/ffmpeg/5.0.1/include/libavformat/avformat.h:2161:41: note: passing argument to parameter 'decoder_ret' here
extras.c:193:13: error: implicit declaration of function 'avfilter_compare_sign_bypath' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
extras.c:193:13: note: did you mean 'lpms_compare_sign_bypath'?
extras.c:191:5: note: 'lpms_compare_sign_bypath' declared here
extras.c:204:13: error: implicit declaration of function 'avfilter_compare_sign_bybuff' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
extras.c:204:13: note: did you mean 'lpms_compare_sign_bybuffer'?
extras.c:202:5: note: 'lpms_compare_sign_bybuffer' declared here
```

Reply:
If tests pass on GH Actions, that's fine. M1 may fail for some unrelated reasons. I also experience some test failures on M1.